samples/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "samples",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
i am not sure if you need a package.json here , we will need to merger the code together or make them as monorepo